CMakeLists.txt

```diff
@@ -463,37 +463,69 @@ ELSE (OPENSSL_FOUND)
     SET_TARGET_PROPERTIES(libh2o-evloop PROPERTIES EXCLUDE_FROM_ALL 1)
 ENDIF (OPENSSL_FOUND)
 
-ADD_CUSTOM_TARGET(lib-examples DEPENDS examples-http1client examples-simple examples-socket-client)
+ADD_CUSTOM_TARGET(lib-examples DEPENDS examples-http1client examples-simple examples-socket-client examples-latency-optimization)
```

Comment:
Please do not add latency-optimization as a dependency to lib-examples. It should instead be added to `lib-examples-evloop` with possibly changing the name to `examples-latency-optimization-evloop` for consistency.

This is because latency optimization only works if evloop is used and `examples-latency-optimization` links against evloop (not libuv).

Reply:
yes, fixed